views/deleteStatsByPlayer.pug

```diff
@@ -0,0 +1,24 @@
+html
+  head
+    title Bruins Stats API
+    link(rel='stylesheet', href='/css/styles.css')
+  body
+    header
+      .name Boston Bruin 2019 Stats API
+    table
+        tr
+          th
+            a.link(href=' /stats') Get/stats
+          th
+            a.link(href=' /statsByPlayer') GET/stats/{player}
+          th
+            a.link(href=' /postStats') POST/stats
+          th
+            a.link(href=' /deleteStatsByPlayer') DELETE/stats/{player}
+    p.Route Method: DELETE/stats/{player}
+      <br> Route: https://localhost:1111/stats/{player}
+      <br> Headers: Authorization: YOUR_API_KEY_HERE
```

Comment:
Will you be requiring an API key for use of your API? Adding auth is a significant amount of work and not something we will cover in class.